app/controllers/assessments_controller.rb

```diff
@@ -133,8 +133,8 @@ def update
     typed_reported_symptoms.each do |symptom|
       new_val = symptom.bool_value
       old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.bool_value
-      unless new_val.nil? || old_val.nil?
-        delta << symptom.name + '=' + (new_val ? 'Yes' : 'No') if new_val != old_val
+      if (!new_val.nil? || !old_val.nil?) && new_val != old_val
```

Comment:
This is confusing, what about `new_val.present? && old_val.present? && new_val != old_val`?